analyze_caliper.py

```diff
@@ -21,20 +21,25 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-f', '--cali_file', help="The cali file to be analyzed should go here")
     parser.add_argument('parallel_type', choices=['raja', 'agency'], help="Parallelism framework being used.")
-    parser.add_argument('application_type', choices=['fft', 'mmult'], help="Application benchmark being plotted")
+    parser.add_argument('application_type', choices=['fft', 'mmult', 'fft2d'], help="Application benchmark being plotted")
     parser.add_argument('-o', '--output_file', help="Name of the output file")
     parser.add_argument('-q', '--cali_query_loc', help="Path to cali query executable")
 
     args = parser.parse_args()
     if args.parallel_type == 'raja' and args.application_type == 'mmult':
-        print args.cali_query_loc
+        plot_raja_mmult(args.cali_file, args.output_file, args.cali_query_loc)
+    elif args.parallel_type == 'raja' and args.application_type == 'fft2d':
         plot_raja_mmult(args.cali_file, args.output_file, args.cali_query_loc)
```

Comment:
I should give this a new name, but it turned out that the two functions can parse it the same way. Didn't have time to generalize it now.